pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -1105,3 +1104,33 @@ func setOVNAnnotations(c *Config, annotations map[string]string) error {
 
 	return nil
 }
+
+func validateProviderNetwork(config *Config, providerNetwork *kubevirttypes.ProviderNetwork) error {
```

Comment:
The function's name doesn't reflect its behavior; while it implies that it performs validation, it actually mutates data.